hbar.go

```diff
@@ -45,15 +49,48 @@ func (hbar Hbar) As(unit HbarUnit) float64 {
 
 func (hbar Hbar) String() string {
 	// Format the string as tinybar if the value is 1000 tinybar or less
-	if hbar.tinybar <= 1000 {
-		return fmt.Sprintf("%v tħ", hbar.tinybar)
+	if -1000 <= hbar.tinybar && hbar.tinybar <= 1000 {
+		return fmt.Sprintf("%v tℏ", hbar.tinybar)
 	}
 
 	return fmt.Sprintf("%v ℏ", float64(hbar.tinybar)/float64(HbarUnits.Hbar.numberOfTinybar()))
 }
 
+func HbarFromString(hbar string) (Hbar, error) {
+	match, err := regexp.Compile(`^((?:\+|\-)?\d+(?:\.\d+)?)(?: (tℏ|μℏ|mℏ|ℏ|kℏ|Mℏ|Gℏ))?$`)
+	if err != nil {
+		return Hbar{}, err
+	}
+
+	matchArray := match.FindStringSubmatch(hbar)
+	if len(matchArray)== 0{
+		return Hbar{}, errors.New("Invalid number and/or symbol.")
+	}
+
+	a, err := strconv.ParseFloat(matchArray[1], 64)
+	if err != nil {
+		return Hbar{}, err
+	}
+
+	if strings.Contains(hbar, "tℏ") {
```

Comment:
Move this into a method `hbarUnitFromString()` and instead of using `strings.Contains()` do a string comparison instead.